src/main/resources/db/migration/V1_2__authorised_by_size_change.sql

```diff
@@ -0,0 +1,2 @@
+alter table non_association
+    alter column authorised_by type varchar(60);
```

Comment:
If we increase this column here because usernames may be longer we should also increase the capacity of `updated_by` and `closed_by` as well as they may have the same problem as they are field used to store usernames as well.

Reply:
No, because they are user names from NOMIS which are 30 chars

Authorised by is a free text field in NOMIS of 60 chars